<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminWidget</class>
 <widget class="QWidget" name="AdminWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1065</width>
    <height>646</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1065</width>
    <height>646</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1065</width>
    <height>646</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btn_exit">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>20</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>注销</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>1021</width>
     <height>591</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab_user">
    <attribute name="title">
     <string>查看用户</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget_user">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>30</y>
       <width>871</width>
       <height>521</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_deleteuser">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>封禁用户</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_modity">
    <attribute name="title">
     <string>查看商品</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget_modity">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>30</y>
       <width>521</width>
       <height>521</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_deletemodity">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>40</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>下架商品</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_modity_name">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_search_name">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>40</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>搜索商品</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget_modity_byname">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>431</width>
       <height>451</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_order">
    <attribute name="title">
     <string>查看订单</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget_order">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>841</width>
       <height>511</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
